app/styles/modules/_password-row.scss

```diff
@@ -95,24 +78,3 @@
   color: $text-color;
   outline: none;
 }
-
-/* this one is impossible to do w/o sibling selector
```

Comment:
I think the only thing that should be removed here is the comment. The CSS is still used to change the "show" button colors depending on which state the password field is in.